Application/EdFi.Ods.AdminApp.Management.Tests/ClaimSetEditor/ClaimSetCheckServiceV6Tests.cs

```diff
@@ -25,9 +25,6 @@ public void ShouldReturnTrueIfRequiredClaimSetsExist()
         };
         Save(testApplication);
 
-        var testAbConnectClaimSet = new ClaimSet { ClaimSetName = CloudsOdsAcademicBenchmarksConnectApp.DefaultClaimSet, Application = testApplication };
-        Save(testAbConnectClaimSet);
```

Comment:
I took out everything related to Learning Standards and ABConnect. I might have gone too far; for example, maybe the claim set above should still be supported? I don't know what it is for. So we might need to restore it.

Reply:
Glad to see this removed out of Admin API.  I believe claim set is baked into ODS/API at this point, fine to leave as we'll still have the CLI tool in the Exchange.  However, as outgoing functionality, no need for test coverage here.